src/environments/environment.ts

```diff
@@ -6,6 +6,7 @@
 export const environment = {
   production: false,
   graphqlUri: 'http://localhost:2020/graphql',
+  restUri: 'http://localhost:4200/assets/json',
```

Comment:
Let's just keep this `/assets/json`? I think it will always work for all environments.